nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -566,6 +566,12 @@
 
 - The `services.patroni.raft` option has been removed, as Raft has been [deprecated by upstream since 3.0.0](https://github.com/patroni/patroni/blob/master/docs/releases.rst#version-300)
 
+- The `jd-cli` package was removed due to an inactive upstream and a dependency on the end-of-life Gradle 6.
```

Comment:
No Gradle dependency here. It’s the dependency on the JCenter repository that was shut down.

Reply:
Oops, fixed.